src/bkl/plugins/vs201x.py

```diff
@@ -75,24 +75,24 @@ class VS201xToolsetBase(VSToolsetBase):
     #: ToolsVersion property
     tools_version = "4.0"
 
-    property_sheets_property_defined = False
+    properties_target_vs = [
```

Comment:
Should this perhaps be called `properties_target_vs201x` for consistency?

Reply:
Yes.